### ISSUE TITLE
Is it possible to add a blank page after the coverpage and before the titlepage?

### ISSUE THREAD

**Issue by guslaet** [nmfs-opensci/quarto_titlepages#38]
The question is objective. I need this because I’m making a scrbook and the coverpage is scrambling with page numbering.

So I would have:

p. 1 = coverpage
p. 2 = blankpage
p. 3 = titlepage
p. 4 = includes1.tex etc.

**Comment by eeholmes** [comment]
Yes, you can include as many static tex files/pages as you need after the coverpage or titlepage. See instructions here:
https://nmfs-opensci.github.io/quarto_titlepages/06-static-files.html#copyright-dedication-and-other-static-pages

Basically I think you just need to create a file like `blank.tex`. I think this should do it but I didn't check:
```
\newpage
\thispagestyle{plain}
\mbox{}
\clearpage
```

**Comment by eeholmes** [comment]
Moving this to discussion Q&A.
